<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFunPointEdit</class>
 <widget class="QWidget" name="CFunPointEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFunPointEdit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_no">
           <property name="text">
            <string>排序编号：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_no"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>条目名称：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_type">
           <property name="text">
            <string>条目类型：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_type"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_plugin">
           <property name="text">
            <string>选择插件：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_plugin">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_icon">
           <property name="text">
            <string>图标操作：</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_auth">
           <property name="text">
            <string>权限管理：</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QTableWidget" name="tableWidget_auth">
           <column>
            <property name="text">
             <string>用户组</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>用户名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>权限</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="7" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnIconImport">
              <property name="text">
               <string>图标更换...</string>
              </property>
              <property name="icon">
               <iconset resource="skgui.qrc">
                <normaloff>:/images/application</normaloff>:/images/application</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnIconExport">
              <property name="text">
               <string>图标导出...</string>
              </property>
              <property name="icon">
               <iconset resource="skgui.qrc">
                <normaloff>:/images/export</normaloff>:/images/export</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_wnd">
           <property name="text">
            <string>绘图名称：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_wnd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>保存(&amp;S)</string>
           </property>
           <property name="icon">
            <iconset resource="skgui.qrc">
             <normaloff>:/images/save</normaloff>:/images/save</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skgui.qrc"/>
 </resources>
 <connections/>
</ui>
